### PR TITLE
[CLDC-1659] new managing agent page

### PR DESCRIPTION
**Pull request by ghost**
https://digital.dclg.gov.uk/jira/browse/CLDC-1658

### CODE REVIEW

File: app/models/bulk_upload.rb [hunk 1]
```diff
@@ -35,10 +35,7 @@ def process(current_user)
           created_by: current_user,
         )
         map_row(row).each do |attr_key, attr_val|
-          update = lettings_log.update(attr_key => attr_val)
-          unless update
-            # TODO: determine what to do when a bulk upload contains field values that don't pass validations
-          end
+          _update = lettings_log.update(attr_key => attr_val)
```
Comment (ghost) [inline app/models/bulk_upload.rb:38]: rubocop changes

File: app/helpers/navigation_items_helper.rb [hunk 2]
```diff
@@ -38,15 +40,17 @@ def secondary_items(path, current_organisation_id)
         NavigationItem.new("Schemes", "/organisations/#{current_organisation_id}/schemes", subnav_supported_housing_schemes_path?(path)),
         NavigationItem.new("Users", "/organisations/#{current_organisation_id}/users", subnav_users_path?(path)),
         NavigationItem.new("About this organisation", "/organisations/#{current_organisation_id}", subnav_details_path?(path)),
-        NavigationItem.new("Housing providers", "/organisations/#{current_organisation_id}/housing-providers", housing_providers_path?(path)),
+        (NavigationItem.new("Housing providers", "/organisations/#{current_organisation_id}/housing-providers", housing_providers_path?(path)) if FeatureToggle.managing_owning_enabled?),
```
Comment (jamesrose) [inline app/helpers/navigation_items_helper.rb]: Can you see why we wouldn't use path helpers consistently here?

File: config/initializers/feature_toggle.rb [hunk 3]
```diff
@@ -8,4 +8,10 @@ def self.sales_log_enabled?
 
     false
   end
+
+  def self.managing_owning_enabled?
+    return true unless Rails.env.production?
```
Comment (jamesrose) [inline config/initializers/feature_toggle.rb:13]: 👍

**Review by jamesrose — APPROVED** [review]
Nice! I just had one comment about inconsistent path helpers, but we can return to it later if you wish.

**Pull request merged**